shm_wrapper/shm_wrapper.c

```diff
@@ -367,30 +370,37 @@ void print_robot_desc() {
     my_sem_post(rd_sem, "robot_desc_mutex (in print)");
 }
 
-void print_gamepad_state() {
-    char** button_names = get_button_names();
-    char** joystick_names = get_joystick_names();
-    uint32_t buttons;
+void print_inputs_state() {
+    uint64_t buttons;
     float joysticks[4];
+    char** button_names[] = {get_button_names(), get_key_names()};
+    int button_lengths[] = {NUM_GAMEPAD_BUTTONS, NUM_KEYBOARD_BUTTONS};
+    char** joystick_names = get_joystick_names();
 
-    printf("Current Gamepad State:\n");
-    if (gamepad_read(&buttons, joysticks) == -1) {
-        printf("\tNo gamepad currently connected\n");
-    } else {
-        printf("\tPushed Buttons:\n");
-        // only print pushed buttons (so we don't print out many lines of output each time we all this function)
-        for (int i = 0; i < NUM_GAMEPAD_BUTTONS; i++) {
-            if (buttons & (1 << i)) {
-                printf("\t\t%s\n", button_names[i]);
+    for (int i = 0; i < 2; i++) {
```

Comment:
Comment about `#define` and 2

Reply:
well its gonna get deleted in your PR so its fine lol